samples/boards/st/hello_world_xip/README.rst

```diff
@@ -0,0 +1,64 @@
+.. _hello_world_xip:
+
+Hello World in XiP
+##################
+
+Overview
+********
+
+A simple sample that can be used with any :ref:`supported board <boards>`
+which has an external NOR octo- or quad- flash and
+prints "Hello World from external flash" to the console.
+The application is built and linked and downloaded in the external flash
+while the mcuboot is built and downloaded for the internal flash
+There is an overlay to set the partition in the external flash
+
+.. code-block:: console
+
+    chosen {
+        zephyr,flash = &mx25lm51245;
+        zephyr,flash-controller = &mx25lm51245;
+    };
+
+
+Building and Running
+********************
+
+This application can be built with **sysbuild** and **SB_CONFIG_BOOTLOADER_MCUBOOT**
+West flash will Download the build/mcuboot/zephyr/zephyr.bin at internal flash address 0x08000000
+and the build/hello_world_xip/zephyr/zephyr.signed.bin at internal flash address 0x90000000 (chosen zephyr,code-partition)
```

Comment:
This is is the same that what is available with `samples/sysbuild/with_mcuboot` I'm not clear on what this new sample is bringing.

Reply:
yes, more or less
Removing this stm32 sample